tests/lib/__init__.py

```diff
@@ -1095,7 +1094,7 @@ def create_basic_sdist_for_package(
     for fname in files:
         path = script.temp_path / fname
         path.parent.mkdir(exist_ok=True, parents=True)
-        path.write_bytes(ensure_binary(files[fname]))
+        path.write_text(files[fname])
```

Comment:
`Path.write_text()` uses platform encoding, but `ensure_binary()` uses UTF-8 by default. So this should be

```python
path.write_bytes(files[fname].encode())
```

instead.

Reply:
I'd suggest being even more explicit, and go with `.encode("utf-8")`.